YogaKit.podspec

```diff
@@ -17,17 +17,23 @@ podspec = Pod::Spec.new do |spec|
   spec.authors = 'Facebook'
   spec.source = {
     :git => 'https://github.com/facebook/yoga.git',
-    :tag => spec.version.to_s,
+    :branch => "master",
```

Comment:
Don't point this to master, otherwise the people using the YogaKit library will always fetch the latest master even if they pin their version.

Reply:
updated, now it point to 1.16.1